doc/classes/Path2D.xml

```diff
@@ -9,9 +9,55 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="is_debug_global_enabled" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if the global path debug is enabled.
+			</description>
+		</method>
+		<method name="set_debug_global_enabled">
+			<return type="void" />
+			<param index="0" name="enabled" type="bool" />
+			<description>
+				If [code]true[/code], enables the global path debug.
+			</description>
+		</method>
+	</methods>
 	<members>
 		<member name="curve" type="Curve2D" setter="set_curve" getter="get_curve">
 			A [Curve2D] describing the path.
 		</member>
+		<member name="debug_custom_color" type="Color" setter="set_debug_custom_color" getter="get_debug_custom_color" default="Color(1, 1, 1, 1)">
+			The custom color used to draw the path debug when [member debug_custom_enabled] is enabled.
+		</member>
+		<member name="debug_custom_enabled" type="bool" setter="set_debug_custom_enabled" getter="get_debug_custom_enabled" default="false">
+			If [code]true[/code], the debug custom properties override the global defaults.
+		</member>
+		<member name="debug_custom_fish_bones_enabled" type="bool" setter="set_debug_custom_fish_bones_enabled" getter="get_debug_custom_fish_bones_enabled" default="true">
+			If [code]true[/code], the debug display adds additional fish bones along the path segments to visualize direction. Used when [member debug_custom_sampling_enabled] is enabled.
+		</member>
+		<member name="debug_custom_fish_bones_interval" type="int" setter="set_debug_custom_fish_bones_interval" getter="get_debug_custom_fish_bones_interval" default="4">
+			The custom interval for adding fish bones to the path sample points. E.g. a value of 4 places a fish bone for every 4-th sampled point. Used when [member debug_custom_sampling_enabled] is enabled.
```

Comment:
The hyphen is not needed, correct me if it's an American English thing or the secret forbidden rule.
```suggestion
			The custom interval used to render fish bones to the points sampled in the path. For example, a value of [code]4[/code] renders a fish bone every 4th sampled point. Used when [member debug_custom_sampling_enabled] is [code]true[/code].
```